src/main/java/net/dv8tion/jda/internal/entities/WebhookImpl.java

```diff
@@ -252,7 +253,12 @@ public WebhookMessageActionImpl<Void> sendRequest()
     {
         checkToken();
         Route.CompiledRoute route = Route.Webhooks.EXECUTE_WEBHOOK.compile(getId(), token);
-        WebhookMessageActionImpl<Void> action = new WebhookMessageActionImpl<>(api, channel, route, (json) -> null);
+
+        //TODO: This MessageChannel cast seems problematic, especially once ForumChannels come along.
+        // Note for Minn: Can we make this cast here because the WebhookClient is only used when responding to interactions
+        //                actually present in a MessageChannel?
+        //                Also might not actually be needed as it doesn't look like WebhookMessageActionImpl actually uses the channel variable?
+        WebhookMessageActionImpl<Void> action = new WebhookMessageActionImpl<>(api, (MessageChannel) channel, route, (json) -> null);
```

Comment:
Yeah, I don't think this channel is used for anything and can be removed.

Also, as far as I know, you can only use interactions in message channels currently anyway. It wouldn't make sense to send a followup if you can't see it in the channel. So the entire concept of InteractionHook only makes sense in message channels in the first place.

Reply:
Done